imagesearch/predict.py

```diff
@@ -75,28 +59,38 @@ def main():
         input_fn=input_fn,
         as_iterable=True)
 
+    # drop out of tensorflow into regular python/numpy
     predictions_list = list(predictions_iter)
     features_length = len(predictions_list[0]['encoded_image'].flatten())
 
-    print("Indexing {} images".format(len(predictions_list)))
+    # flatten embeddings so they play nice with annoy
+    embeddings = np.zeros(shape=(len(predictions_list), 8192), dtype=np.float64)
```

Comment:
Don't you have the 8192 value as a property in the metadata? Would be better not to hard-code.